lib/Twig/SourceContextLoaderInterface.php

```diff
@@ -9,6 +9,13 @@
  * file that was distributed with this source code.
  */
 
+/**
+ * Adds a getSourceContext() method for loaders.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @deprecated since 1.27 (to be removed in 3.0)
```

Comment:
Not really. Notice that there is no deprecation notice triggered when you are using this interface. That's the same as the `Twig_ExistsLoaderInterface`.